xmake-rules.lua

```diff
@@ -173,7 +173,7 @@ rule("commonlibob64.plugin")
         local root_dir = path.join(os.tmpdir(), "packages", project.name() or "", target:name())
         os.tryrm(root_dir)
 
-        local srcfiles, dstfiles = target:installfiles(path.join(root_dir, "Data"))
+        local srcfiles, dstfiles = target:installfiles(path.join(root_dir, "OblivionRemastered/Binaries/Win64/"))
```

Comment:
```suggestion
        local srcfiles, dstfiles = target:installfiles(root_dir)
```

and on lines 71 and 72 change:
```
        target:add("installfiles", target:targetfile(), { prefixdir = "OBSE/Plugins" })
        target:add("installfiles", target:symbolfile(), { prefixdir = "OBSE/Plugins" })
```
to
```
        target:add("installfiles", target:targetfile(), { prefixdir = "OblivionRemastered/Binaries/Win64/OBSE/Plugins" })
        target:add("installfiles", target:symbolfile(), { prefixdir = "OblivionRemastered/Binaries/Win64/OBSE/Plugins" })
```

Reply:
This does mean you'll have to change the environment variable to point to the root game install, but that was my original intention